cookbooks/aws-parallelcluster-environment/resources/nfs/nfs_ubuntu22+.rb

```diff
@@ -29,11 +29,11 @@
   action_install_nfs
   action_install_nfs4
   action_disable_start_at_boot
-  node.default['nfs']['config']['server_template'] = '/etc/nfs.conf.d/parallelcluster-nfs.conf'
 end
 
 action_class do
   def override_server_template
+    node.default['nfs']['config']['server_template'] = '/etc/nfs.conf.d/parallelcluster-nfs.conf'
```

Comment:
nice catch!

Reply:
Probably the alternative was to move this override outside of the action class, to be "global".